### ISSUE TITLE
How to use pg_cron for Postgresql13

### ISSUE THREAD

**Issue by LegolasM** [citusdata/pg_cron#114]
Hello!
As writed  v.1.3.0 has PostgreSQL 13 support.
How can I use pg_cron in Postgresql13?
It works only on 12.

```
$ sudo dnf search pg_cron
Last metadata expiration check: 0:10:05 ago on Fri 23 Oct 2020 11:00:46 AM MSK.
================================================================================== Name Matched: pg_cron ==================================================================================
pg_cron_12.x86_64 : Run periodic jobs in PostgreSQL
pg_cron_11.x86_64 : Run periodic jobs in PostgreSQL
pg_cron_10.x86_64 : Run periodic jobs in PostgreSQL
pg_cron_96.x86_64 : Run periodic jobs in PostgreSQL
pg_cron_95.x86_64 : Run periodic jobs in PostgreSQL

$ sudo dnf install pg_cron_12.x86_64
citusdata_community                                                                                                                                        848  B/s | 1.0 kB     00:01
citusdata_community-source                                                                                                                                 1.8 kB/s | 951  B     00:00
Dependencies resolved.
===========================================================================================================================================================================================
 Package                                             Architecture                           Version                                           Repository                              Size
===========================================================================================================================================================================================
Installing:
 pg_cron_12                                          x86_64                                 1.3.0-1.rhel8                                    pgdg12                                  96 k
Installing dependencies:
 postgresql12                                        x86_64                                 12.4-1PGDG.rhel8                                  pgdg12                                 1.6 M
 postgresql12-server                                 x86_64                                 12.4-1PGDG.rhel8                                  pgdg12                                 5.2 M
```

Thank you.
All the best

**Comment by nonara** [comment]
@LegolasM Try building from source. See instructions: https://github.com/citusdata/pg_cron#installing-pg_cron

**Comment by paisleyrob** [comment]
We built it from source against postgres 13, so I know it works. Here's the top of our `Dockerfile`, note the `sed` command corrects a build warning that's been corrected in `master` but they haven't made a release yet

```Dockerfile
FROM docker.io/postgis/postgis:13-3.1-alpine@sha256:780464aed283f3ccbb8a09934188610526fb8d87939b2fdae6469724a3ec5635 AS build
ADD https://github.com/citusdata/pg_cron/archive/v1.3.0.tar.gz /tmp
WORKDIR /tmp
RUN tar -xf v1.3.0.tar.gz
WORKDIR /tmp/pg_cron-1.3.0
RUN apk add --no-cache \
        clang=10.0.0-r2 \
        gcc=9.3.0-r2 \
        libc-dev=0.7.2-r3 \
        llvm10=10.0.0-r2 \
        make=4.3-r0 \
    && sed -i -e 's/\(-Werror\)/\1 -Wno-cpp/' Makefile \
    && make \
    && make install
```

**Comment by hanefi** [comment]
We released 1.3.0 packages compatible with PG13 about a month ago. However you are using alpine images, and we do not ship packages for alpine.

If you look at the steps for alpine PostGIS images, you can see that they build the PostgreSQL from source as well. See https://registry.hub.docker.com/layers/postgis/postgis/13-3.1-alpine/images/sha256-6d6c721c3b3f4d50045c01d75c38dee491ae149e79115be13d133615a8908d45
